os/board/rtl8730e/src/component/mbed/targets/hal/rtl8730e/us_ticker.c

```diff
@@ -63,7 +63,7 @@ uint32_t us_ticker_read(void)
 	uint64_t us_tick;
 
 	tick_cnt = SYSTIMER_TickGet(); //up counter
-	us_tick = tick_cnt * (1000000 / 32768);
+	us_tick = (uint64_t) (tick_cnt * (1000000 / 32768));
```

Comment:
Is this really working? Not `(uint64_t)tick_cnt × (1000000 / 32768)`?

Reply:
Hi @sunghan-chang ,

I apologize for the mistake. You're correct.
After verified :
The suitable way is to cast the tick_cnt to uint64_t.

![image](https://github.com/Samsung/TizenRT/assets/94035727/f0b5ef47-c747-494f-9467-acd69caf18af)

Done git push --force the changes.
Thank You.